lib/appdirs/__init__.py

```diff
@@ -0,0 +1 @@
+from .appdirs import user_cache_dir
```

Comment:
Even though it doesnt look as clean, the reason i prefer to mimic site packages is it makes updating the libraries easier (create a clean venv, update all packages and copy relevant files/folders to libs).  It makes utilization of it no different than if it was pip-installed.  It requires no maintenance/updates when the package changes.

For example, right now you have only `user_cache_dir` exposed.  If someone sees we have the lib installed they may want to leverage `user_log_dir`.  Currently you can't do that cleanly.  The same applies to attempting to catch `AppDirsError`.  For example you can do the following:
```python
from appdirs.appidrs import AppDirsError
```
...but then its implemented differently than if it was pip installed.